client/checkout/woopay/email-input-iframe.js

```diff
@@ -277,9 +270,9 @@ export const handlePlatformCheckoutEmailInput = async (
 
 	// Store if the subscription login error is being shown
 	// to remove it when change the e-mail address.
-	let hasPlatformCheckoutSubscriptionLoginError = false;
+	let hasWooPaySubscriptionLoginError = false;
```

Comment:
I've noticed that we use `wooPay` and `woopay` interchangeably now. Could we align these two and have the unified approach? After going through lines `216`, `249` and other similar, I would expect this variable to be called 

```suggestion
	let hasWoopaySubscriptionLoginError = false;
```